frontend/ui/src/app/tutorials/agents/agents-tutorial.component.html

```diff
@@ -0,0 +1,264 @@
+                        class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
+                    </span>
+                    <h3 class="text-lg font-medium leading-tight">Fork the hello-distr Application</h3>
+                    <p class="mt-1 text-xs">
+                      Fork the previously deployed
+                      <a
+                        href="https://github.com/glasskube/hello-distr"
+                        class="text-gray-600 dark:text-gray-400 underline"
+                        target="_blank"
+                        >hello-distr application</a
+                      >
+                      on GitHub. You will use this fork to release a new version of your own hello-distr, and then sync
+                      this release into Distr.
+                    </p>
+                    @if (releaseFormGroup.controls.forkDone.touched && releaseFormGroup.controls.forkDone.invalid) {
+                      <div class="mt-2 text-sm text-red-500">
+                        Please mark this as checked when you are done with it.
+                      </div>
+                    }
+                  </li>
+
+                  <li class="mb-10 ms-8">
+                    <span class="absolute flex items-center justify-center -start-2">
+                      <input
+                        type="checkbox"
+                        [formControl]="releaseFormGroup.controls.releaseDone"
+                        class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
+                    </span>
+                    <h3 class="text-lg font-medium leading-tight">Push a hello-distr version to Distr</h3>
+                    <p class="mt-1 text-xs">
+                      The hello-distr README describes how all the pieces fit together. Experiment on your own and try
+                      to get to a state, where a release of your forked repository will create a new application version
+                      in Distr.
+                    </p>
+                    <p class="mt-1 text-xs">
+                      If you need any help or in case you have questions, don't hesitate to ask in
+                      <a
+                        href="https://discord.gg/6qqBSAWZfW"
+                        target="_blank"
+                        class="text-gray-600 dark:text-gray-400 hover:underline"
+                        >Discord</a
+                      >!
+                    </p>
+                    <p class="mt-1 text-xs">
+                      When you are done, you can show us by dropping your hello-distr fork in the text field below:
```

Comment:
Maybe change to "URL of your hello-distr fork"? I think "dropping" a fork could be confusing